frontend/src/contexts/wishlist-context.tsx

```diff
@@ -12,123 +13,60 @@ import {
   likedCars: undefined,
   handleLikeClick: () => undefined,
-  notifications: undefined,
-  clearNotification: () => undefined,
-  undoDelete: () => undefined,
 });
 
-interface WishListNotification {
-  modelId?: string;
-  complectationId?: string;
-  createdAt?: string;
-  carName?: string;
-  timestamp: number;
-}
-
 const WishlistContextProvider: React.FC<{ children: ReactNode }> = ({
   children,
 }) => {
-  const [notifications, setNotifications] = useState<WishListNotification[]>(
-    [] as WishListNotification[],
-  );
-
-  const [deletedCar, setDeletedCar] = useState<WishlistInput>();
   const { data: likedCars } = useGetWishlistEntriesQuery();
 
   const [createWishlist] = useCreateWishlistMutation();
   const [deleteWishlist] = useDeleteWishlistMutation();
 
-  const handleCreateWishlist = async (data: WishlistInput): Promise<void> => {
-    await createWishlist(data);
-  };
-
-  const handleDeleteWishlist = async (data: WishlistInput): Promise<void> => {
-    await deleteWishlist(data);
-    setDeletedCar(data);
-  };
-
   const handleLikeClick = (data: WishlistInput): void => {
     const carId = data.modelId ? data.modelId : data.complectationId;
     const isLiked = likedCars?.includes(carId as string);
 
     isLiked ? handleDeleteWishlist(data) : handleCreateWishlist(data);
-
-    if (isLiked) {
-      const existing = notifications.find(
-        (n) =>
-          n.modelId === data.modelId &&
-          n.complectationId === data.complectationId,
-      );
-      const nextNotifications = existing
-        ? notifications.map((n) =>
-            n.modelId === data.modelId ||
-            n.complectationId === data.complectationId
-              ? { ...existing, ...data }
-              : n,
-          )
-        : notifications.concat({
-            modelId: data.modelId,
-            complectationId: data.complectationId,
-            createdAt: data.createdAt,
-            carName: data.carName,
-            timestamp: new Date().getTime(),
-          });
-      setNotifications(nextNotifications);
-    }
   };
 
-  const handleUndoDelete = (data: WishlistInput): void => {
-    clearNotification(data.modelId ?? data.complectationId ?? uuid4());
-    handleCreateWishlist(data);
+  const handleUndoDelete = async (data: WishlistInput): Promise<void> => {
+    await createWishlist(data);
   };
 
-  const undoDelete = (data: WishlistInput): void => {
-    deletedCar && handleUndoDelete(data);
-  };
+  const handleCreateWishlist = async (data: WishlistInput): Promise<void> => {
+    await createWishlist(data);
 
-  const clearNotification = (id: string | string[]): void => {
-    if (!id) {
-      setNotifications([]);
-    } else {
-      const ids = Array.isArray(id) ? id : [id];
-      const nextNotifications = notifications.filter(
-        ({ modelId, complectationId }) =>
-          !ids?.includes(modelId || complectationId || uuid4()),
-      );
-      setNotifications(nextNotifications);
-    }
+    toast.info(
```

Comment:
In case a user is unauth, he will be redirected to sign-in but the notification still appears. 

I think we need to put this call inside of the `createWishlist` promise using `unwrap().then()`